index.html

```diff
@@ -41,6 +41,30 @@
           </li>
         </ul>
       </div>
+      <div class="desktop_toolbar">
+        <div class="contact_bar">
+          <i class="fa fa-facebook" aria-hidden="true"></i>
+          <i class="fa fa-twitter" aria-hidden="true"></i>
+          <h6>English</h6>
+          <h6>My page</h6>
+          <h6>Log out</h6>
+        </div>
+        <div class="desktop_navbar">
+          <img
+            class="menu_logo"
+            src="./images/CC_Global_Summit.png"
+            alt="Company logo located at the navbar"
+          />
+          <div class="page_bar">
+            <h4><a href="./about.html">About</a></h4>
+            <h4><a href="#Main programs">Programs</a></h4>
+            <h4>Join</h4>
+            <h4>Sponsor</h4>
+            <h4>News</h4>
+          </div>
+          <h5 class="logo_heading">CC Campaign</h5>
+        </div>
```

Comment:
- [ ] Good job implementing the required design 👍 your project looks great and it is very responsive 💯 but there is a little issue with your desktop navbar. It would be great if you could add some spacing between your logo and your links so as to follow the design.
- [ ] Also you'll notice that the `about` link has a different color from the rest of the links and also the `cc campaign` button has a border. It would be great if you could implement those features.
- Your design:
![f2](https://user-images.githubusercontent.com/85113567/152421778-cb7e3518-7066-4457-8777-99024534fa25.PNG)
- Required design:
![f1](https://user-images.githubusercontent.com/85113567/152421812-f21017af-4b4a-4f9e-8dd2-61534ffcf4fb.PNG)

Reply:
Thanks it was very helpful and very nice of you to point it out. 😉 💯